src/conformance.md

```diff
@@ -0,0 +1,292 @@
+errors.  For example, we will attempt to check as a mesh variable any variable that _either_ has a `cf_role` of `mesh_topology`
+_or_ is named in the `mesh` attribute of a data variable or location index set variable.
+
+[^3]: "valid" element indices are values within the range of the corresponding element dimension of the parent mesh,
+allowing for any `start_index` offset.
+
+[^4]: For now at least, we _exclude_ mention of full-3d meshes, whose status is currently in question : 
+cf. [this GitHub discussion][volume_status], as this would add considerable complexity here.
+
+[^5]: For now, we _allow_ 0-d meshes (those with nodes only).  Though, this is not explicitly mentioned in the UGRID v1.0 pages.
+
+[^6]: It is not supported to have boundaries in a 1-D mesh (I.E. one with only edges), though this is not logically
+impossible.  If a specific need arises, this could change in future.
+
+[^7]: Since there is no provision for a `boundary_dimension` attribute, the boundary dimension is always the
+_first_ dimension of a `boundary_node_connectivity`, and its second dimension must always be 2
+(i.e. it cannot be transposed).  If a specific need arises, this could change in future.
+
+[^8]: Requirements on _values_ of variables may not always be practical to check,
+since this requires loading all of their data.
+
+[^9]: The form of the global `Conventions` attribute is determined by the [CF conformance pages][cf_conformance],
+section 2.6.1.  However, the definition of UGRID release numbers is not yet finalised.
+
+[^10]: Providing stable statement references allows conformance statements to evolve without invalidating earlier
+versions of a checker program.  We also provide a linkable web anchor for each one.
+
+[^11]: if a mesh does not share any element dimensions with other meshes, it follows that it also will not share any
+  element coordinates or connectivities with other meshes.
+
+[^12]: invalid 'lookalike' attributes, which are not defined by UGRID, include such as :
+  "node_face_connectivity", "boundary_coordinates", "boundary_dimension" or "node_dimension".
+
+[^13]: for checking purposes, the identity of a connectivity is actually defined by the referring attribute in the parent mesh.
+   However, if the connectivity is correctly formed, then its `cf_role` value will always be the same as this  -- cf:[R303](#R303).
+
+[^14]: in the present [latest CF version][cf_current_version] the only `cf_role` terms defined are the discrete-geometry
+  types : `timeseries_id`, `profile_id` and `trajectory_id`
+
+
+[volume_status]: https://github.com/ugrid-conventions/ugrid-conventions/issues/53#issuecomment-832138564
+[cf_conformance]: http://cfconventions.org/cf-conventions/conformance.html "CF Conventions Conformance pages"
+[cf_conventions]: https://cfconventions.org/ "CF metadata conventions"
+[cf_current_version]: https://cfconventions.org/Data/cf-conventions/cf-conventions-1.9/cf-conventions.html "CF version 1.9"
```

Comment:
Update to 1.10 ?

Reply:
I think if you make the changes to your own fork, the will automatically show up in the PR. But I'll take a quick look right now, and at least approve the typo fixes :-)

Reply:
I guess this should now say "1.11".
Will fix ...